CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [2.10.2] - 2022-11-29
```

Comment:
You forgot to add the diff link. There should be a list around the mid of the file

Reply:
ah I see, there wasn't one for the previous change, should I include those changes in this one?

Reply:
if you can would be great!